docs/reference/manifest/extendedoverrides.md

```diff
@@ -31,10 +31,21 @@ Specifies the full URLs for JSON-formatted files that extend the manifest.
 
 |Attribute|Description|
 |:-----|:-----|
-|Url (required)| The full URL of the extended overrides JSON file. This could be a URL template that uses tokens defined by the [Tokens](tokens.md) element.|
+|Url (required)| The full URL of the extended overrides JSON file. In the future, this value could be a URL template that uses tokens defined by the [Tokens](tokens.md) element. See [Examples](#examples)|
```

Comment:
@Rick-Kirkham I think the last sentence should end with a period i.e. "See [Examples](#examples)."

Reply:
done